rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -119,20 +149,20 @@ object RholangCLI {
     repl(runtime)
   }
 
-  def processFile(conf: Conf, runtime: Runtime[Task], fileName: String)(
+  def processFile(conf: Conf, runtime: Runtime[Task], fileName: String, quiet: Boolean)(
       implicit scheduler: Scheduler
-  ): Unit = {
-    val processTerm: Par => Unit =
+  ): Try[Unit] = {
+    val processTerm: Par => Try[Unit] =
       if (conf.binary()) writeBinary(fileName)
       else if (conf.text()) writeHumanReadable(fileName)
-      else evaluatePar(runtime)
-
-    val source = reader(fileName)
-
-    Interpreter[Coeval]
-      .buildNormalizedTerm(source)
-      .runAttempt
-      .fold(_.printStackTrace(Console.err), processTerm)
+      else evaluatePar(runtime, quiet)
+
+    Try(reader(fileName)).map(source => {
```

Comment:
```suggestion
Try(reader(fileName)).flatMap(source => {
```

Reply:
Otherwise the error handling is 'swallow the exception', because `Try(whatever).map(Failure(_))` is `Success(Failure(whatever))`, which is a "'success'"